.github/workflows/camunda-helm-integration.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         tasklist:
           image:
             tag: SNAPSHOT
+        optimize:
+          image:
+            tag: SNAPSHOT
```

Comment:
didn't @OmranAbazid change this to C8-SNAPSHOT?

Reply:
It is still on a PR here:
https://github.com/camunda/camunda/pull/20614
You can merge this and i will adjust my PR